app/src/gui/components/autoincrement/edit-form.tsx

```diff
@@ -207,20 +208,25 @@ type IncremenenterRangeProps = {
  * @param props component props
  */
 const IncrementerRange = (props: IncremenenterRangeProps) => {
-  const [start, setStart] = useState(props.range.start);
-  const [stop, setStop] = useState(props.range.stop);
+  const [start, setStart] = useState<number | string>(props.range.start);
+  const [stop, setStop] = useState<number | string>(props.range.stop);
```

Comment:
do they really need to be a number or a string? Might be better to on each render parse the value as a number - just choose one?

Reply:
This is to allow for the empty string, if I don't, you can't delete the number that's there to enter a new one.  Note this is only for the field value, not for the final data structure which is a number.